### DISCUSSION TITLE
I cannot see whole name of resource in config map

### DISCUSSION THREAD

**Discussion by LadaVarga** [argoproj/argo-cd#17585] [Ideas]
Hello,
Have you changed something recently in ArgoCD UI, I cannot see the whole name of name of the configmap in configmap view? We use version v2.10.3+0fd6344. I definitely use that table pretty heavily and I was able to the almost whole name of the resource.
![2024-03-21 16_58_49-_ Channel_ team-software-delivery - Procore Technologies, Inc - Slack](https://github.com/argoproj/argo-cd/assets/22859476/0165352b-4de3-4a46-9d4e-db81096ed0d6)
Thanks

**Comment by jannfis** [comment]
Have you upgraded from a previous (<= 2.10) version?

IIRC, the columns never could be resized. The empty space to the right looks suspicious, though.

If you upgraded, it's probably worth a bug issue instead of a discussion.
